File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/drgs/DRG9040U01GrdOrderOutInfo.java
package nta.med.data.model.ihis.drgs;

import java.sql.Date;
import java.sql.Timestamp;

public class DRG9040U01GrdOrderOutInfo {
	private Double drgBunho;                                                                       
    private String bunho;                                                                           
    private Timestamp orderDate;                                                                      
    private Timestamp jubsuDate;                                                                      
    private String jubsuTime;                                                                      
    private String doctor;                                                                          
    private String doctorName;                                                                     
    private String gwa;                                                                             
    private String buseoName;                                                                      
    private Timestamp actDate;                                                                        
    private String actYn;                                                                          
    private Timestamp sunabDate;                                                                      
    private Timestamp chulgoDate;                                                                     
    private String boryuYn;                                                                        
    private String orderRemark;                                                                    
    private String drgRemark;                                                                      
    private String wonyoiOrderYn;
	public DRG9040U01GrdOrderOutInfo(Double drgBunho, String bunho,
			Timestamp orderDate, Timestamp jubsuDate, String jubsuTime,
			String doctor, String doctorName, String gwa, String buseoName,
			Timestamp actDate, String actYn, Timestamp sunabDate,
			Timestamp chulgoDate, String boryuYn, String orderRemark,
			String drgRemark, String wonyoiOrderYn) {
		super();
		this.drgBunho = drgBunho;
		this.bunho = bunho;
		this.orderDate = orderDate;
		this.jubsuDate = jubsuDate;
		this.jubsuTime = jubsuTime;
		this.doctor = doctor;
		this.doctorName = doctorName;
		this.gwa = gwa;
		this.buseoName = buseoName;
		this.actDate = actDate;
		this.actYn = actYn;
		this.sunabDate = sunabDate;
		this.chulgoDate = chulgoDate;
		this.boryuYn = boryuYn;
		this.orderRemark = orderRemark;
		this.drgRemark = drgRemark;
		this.wonyoiOrderYn = wonyoiOrderYn;
	}
	public Double getDrgBunho() {
		return drgBunho;
	}
	public void setDrgBunho(Double drgBunho) {
		this.drgBunho = drgBunho;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public Timestamp getOrderDate() {
		return orderDate;
	}
	public void setOrderDate(Timestamp orderDate) {
		this.orderDate = orderDate;
	}
	public Timestamp getJubsuDate() {
		return jubsuDate;
	}
	public void setJubsuDate(Timestamp jubsuDate) {
		this.jubsuDate = jubsuDate;
	}
	public String getJubsuTime() {
		return jubsuTime;
	}
	public void setJubsuTime(String jubsuTime) {
		this.jubsuTime = jubsuTime;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getBuseoName() {
		return buseoName;
	}
	public void setBuseoName(String buseoName) {
		this.buseoName = buseoName;
	}
	public Timestamp getActDate() {
		return actDate;
	}
	public void setActDate(Timestamp actDate) {
		this.actDate = actDate;
	}
	public String getActYn() {
		return actYn;
	}
	public void setActYn(String actYn) {
		this.actYn = actYn;
	}
	public Timestamp getSunabDate() {
		return sunabDate;
	}
	public void setSunabDate(Timestamp sunabDate) {
		this.sunabDate = sunabDate;
	}
	public Timestamp getChulgoDate() {
		return chulgoDate;
	}
	public void setChulgoDate(Timestamp chulgoDate) {
		this.chulgoDate = chulgoDate;
	}
	public String getBoryuYn() {
		return boryuYn;
	}
	public void setBoryuYn(String boryuYn) {
		this.boryuYn = boryuYn;
	}
	public String getOrderRemark() {
		return orderRemark;
	}
	public void setOrderRemark(String orderRemark) {
		this.orderRemark = orderRemark;
	}
	public String getDrgRemark() {
		return drgRemark;
	}
	public void setDrgRemark(String drgRemark) {
		this.drgRemark = drgRemark;
	}
	public String getWonyoiOrderYn() {
		return wonyoiOrderYn;
	}
	public void setWonyoiOrderYn(String wonyoiOrderYn) {
		this.wonyoiOrderYn = wonyoiOrderYn;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Drgs/DRG9040U01GrdOrderOutInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Drgs
{
    [Serializable]
	public class DRG9040U01GrdOrderOutInfo
	{
		private String _drgBunho;
		private String _bunho;
		private String _orderDate;
		private String _jubsuDate;
		private String _jubsuTime;
		private String _doctor;
		private String _doctorName;
		private String _gwa;
		private String _buseoName;
		private String _actDate;
		private String _actYn;
		private String _sunabDate;
		private String _chulgoDate;
		private String _boryuYn;
		private String _orderRemark;
		private String _drgRemark;
		private String _wonyoiOrderYn;
		private String _dataRowState;

		public String DrgBunho
		{
			get { return this._drgBunho; }
			set { this._drgBunho = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String OrderDate
		{
			get { return this._orderDate; }
			set { this._orderDate = value; }
		}

		public String JubsuDate
		{
			get { return this._jubsuDate; }
			set { this._jubsuDate = value; }
		}

		public String JubsuTime
		{
			get { return this._jubsuTime; }
			set { this._jubsuTime = value; }
		}

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String BuseoName
		{
			get { return this._buseoName; }
			set { this._buseoName = value; }
		}

		public String ActDate
		{
			get { return this._actDate; }
			set { this._actDate = value; }
		}

		public String ActYn
		{
			get { return this._actYn; }
			set { this._actYn = value; }
		}

		public String SunabDate
		{
			get { return this._sunabDate; }
			set { this._sunabDate = value; }
		}

		public String ChulgoDate
		{
			get { return this._chulgoDate; }
			set { this._chulgoDate = value; }
		}

		public String BoryuYn
		{
			get { return this._boryuYn; }
			set { this._boryuYn = value; }
		}

		public String OrderRemark
		{
			get { return this._orderRemark; }
			set { this._orderRemark = value; }
		}

		public String DrgRemark
		{
			get { return this._drgRemark; }
			set { this._drgRemark = value; }
		}

		public String WonyoiOrderYn
		{
			get { return this._wonyoiOrderYn; }
			set { this._wonyoiOrderYn = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public DRG9040U01GrdOrderOutInfo() { }

		public DRG9040U01GrdOrderOutInfo(String drgBunho, String bunho, String orderDate, String jubsuDate, String jubsuTime, String doctor, String doctorName, String gwa, String buseoName, String actDate, String actYn, String sunabDate, String chulgoDate, String boryuYn, String orderRemark, String drgRemark, String wonyoiOrderYn, String dataRowState)
		{
			this._drgBunho = drgBunho;
			this._bunho = bunho;
			this._orderDate = orderDate;
			this._jubsuDate = jubsuDate;
			this._jubsuTime = jubsuTime;
			this._doctor = doctor;
			this._doctorName = doctorName;
			this._gwa = gwa;
			this._buseoName = buseoName;
			this._actDate = actDate;
			this._actYn = actYn;
			this._sunabDate = sunabDate;
			this._chulgoDate = chulgoDate;
			this._boryuYn = boryuYn;
			this._orderRemark = orderRemark;
			this._drgRemark = drgRemark;
			this._wonyoiOrderYn = wonyoiOrderYn;
			this._dataRowState = dataRowState;
		}

	}
}